internal/server/server.go

```diff
@@ -111,11 +128,10 @@ func RunServer(config *Config) error {
 			otelgrpc.StreamServerInterceptor(),
 		)),
 		grpc.UnaryInterceptor(grpc_middleware.ChainUnaryServer(
-			grpc_logrus.UnaryServerInterceptor(logrusEntry),
 			authInterceptor,
+			grpc_logrus.UnaryServerInterceptor(logrusEntry),
```

Comment:
is there any reason to put this after the auth interceptor? if none we can put it back like previous order as it was